core/ir/aarch64/instr_create_api.h

```diff
@@ -739,7 +740,11 @@
  * \param Rn   The input register containing the virtual address to use.
  *             No alignment restrictions apply to this VA.
  */
-#define INSTR_CREATE_dc_civac(dc, Rn) instr_create_0dst_1src(dc, OP_dc_civac, Rn)
+#define INSTR_CREATE_dc_civac(dc, Rn)                                                   \
+    instr_create_0dst_1src(dc, OP_dc_civac,                                             \
+                           opnd_create_base_disp_aarch64(opnd_get_reg(Rn), DR_REG_NULL, \
+                                                         0, false, 0, 0,                \
+                                                         OPSZ_CACHE_LINE()))
```

Comment:
I see that there are `OPSZ_*` variables for 32, 64, 128 and 512. So we don't need to define any others that could be a possible cache line size on some systems.